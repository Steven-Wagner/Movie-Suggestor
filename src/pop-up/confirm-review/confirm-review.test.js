import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import ConfirmReviewPopUp from './confirm-review';

describe('<ConfirmReviewPopUp/>', () => {
    it('renders without crashing', () => {
    const div = document.createElement('div');

    ReactDOM.render(
        <Router>
            <ConfirmReviewPopUp user= "1" movie= "Test Movie" hidden="false"/>
        </Router>,
        div);
    ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the UI as expected', () => {
        const tree = renderer
        .create(
            <Router>
                <ConfirmReviewPopUp user= "1" movie= "Test Movie" hidden="false"/>
            </Router>)
        .toJSON();
        expect(tree).toMatchSnapshot();  
    });
});