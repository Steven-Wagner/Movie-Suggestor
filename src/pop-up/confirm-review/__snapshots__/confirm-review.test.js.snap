// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConfirmReviewPopUp/> renders the UI as expected 1`] = `
<div
  className="confirm-review-popup"
  hidden={false}
>
  <p>
    Would you like to review 
    Test Movie
  </p>
  <button
    className="remote-button"
    onClick={[Function]}
    type="button"
  >
    Review It
  </button>
  <button
    className="remote-button"
    onClick={[Function]}
    type="button"
  >
    Maybe Later
  </button>
</div>
`;
